outputs.tf

```diff
@@ -3,9 +3,9 @@ output "vpc_id" {
 }
 
 output "public_subnets" {
```

Comment:
I always go back and forth on this point. I struggle with calling it `_ids` (which is ugly, but clear) or without the suffix, which looks good, but maybe implies an object (of which is not a valid resource type to pass as an input). Should we rename it in `tf_subnets` to drop the `_ids`?

Reply:
Rename `public_subnets` to `public_subnet_ids`